chrono/include/cocaine/chrono.hpp

```diff
@@ -69,6 +69,7 @@ class chrono_t:
         streamed<io::timer_id_t>
         set_timer_impl(double first, double repeat, bool send_id);
 
+        static constexpr uint64_t minimal_interval_us = 10000;
```

Comment:
it's not constexpr. Besides there is a lot of things to improve (we are using doubles for times).